### DISCUSSION TITLE
Show all other recipients for incoming mail

### DISCUSSION THREAD

**Discussion by hey-zane** [anonaddy/anonaddy#592] [Feature Requests]
I'm on a few group mailing lists, but when I receive mail at my alias it only shows me as the recipient. 
Also, if a message is auto-forwarded to me from another address, it only shows the anonaddy address that it ended up with. 
I'd like to be able to see all the recipients for every incoming message, including my own address please.

**Comment by willbrowningme** [comment]
You can check the following headers:

```
X-AnonAddy-Original-Cc
X-AnonAddy-Original-To
```
